dfmoco2ur-ui/src/components/LoggerFilter.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    maxWidth: 752,
+  },
+  demo: {
+    backgroundColor: theme.palette.background.paper,
+  },
+  title: {
+    margin: theme.spacing(4, 0, 2),
+  },
+}));
+
+function generate(element) {
+  return [0, 1, 2].map((value) =>
+    React.cloneElement(element, {
+      key: value,
+    }),
+  );
+}
+
+export default function LoggerFilter() {
+  const classes = useStyles();
+  const [dense, setDense] = React.useState(false);
```

Comment:
Ich würde nicht einen Boolean als State verwenden. Sondern z.B.
```suggestion
  const [activeFilters, setActiveFilters] = React.useState({Critical: true, Error: true, Warning: true, Info: false, Debug: false});
```
Dann musst du halt immer das ganze Objekt updaten. Vielleicht funktioniert eines der beiden zum dann einen Wert zu aktualisieren:
```js
setActiveFilters({...activeFilters, Info: true});

setActiveFilters(Object.assign(activeFilters, {Info: true});